ci-operator/config/openshift/terraform-providers/openshift-terraform-providers-main.yaml

```diff
@@ -0,0 +1,58 @@
+base_images:
+  base:
+    name: "4.15"
+    namespace: ocp
+    tag: base
+  ocp_builder_rhel-8-golang-1.20-openshift-4.15:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.20-openshift-4.15
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: rhel-8-release-golang-1.20-openshift-4.15
+images:
+- dockerfile_path: Dockerfile
+  from: base
+  inputs:
+    ocp_builder_rhel-8-golang-1.20-openshift-4.15:
+      as:
+      - registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.20-openshift-4.15
+  to: terraform-providers
+promotion:
+  name: "4.15"
+  namespace: ocp
+  tag_by_commit: true
+releases:
+  initial:
+    integration:
+      name: "4.15"
+      namespace: ocp
+  latest:
+    integration:
+      include_built_images: true
+      name: "4.15"
+      namespace: ocp
+resources:
+  '*':
+    requests:
+      cpu: "5"
+      memory: 5Gi
+tests:
+- as: security
+  optional: true
+  steps:
+    env:
+      ALL_PROJECTS: "true"
```

Comment:
Looks like we should delete this line based on the rehearsal:

```
 The following option combination is not currently supported: project-name + all-projects 
```

Reply:
We cannot delete this, otherwise `snyk` won't go into the subdirs and we get 
```
Could not detect supported target files in /go/src/github.com/openshift/terraform-providers.
Please see our documentation for supported languages and target files: https://snyk.co/udVgQ and make sure you are in the right directory. 
```
https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/44608/rehearse-44608-pull-ci-openshift-terraform-providers-main-security/1717979594549628928

Let me try removing the project-name

Reply:
@patrickdillon so it gave me that same message (project-name + all-projects) before but it scanned the codebase and produced an output: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/44608/rehearse-44608-pull-ci-openshift-terraform-providers-main-security/1718003488371249152
I'll bump the timeout.

Reply:
@patrickdillon it's still complaining about `The following option combination is not currently supported: project-name + all-projects` even though I'm not setting `PROJECT_NAME` but the report was generated https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/44608/rehearse-44608-pull-ci-openshift-terraform-providers-main-security/1719106701514248192